ui/pages/routes/routes.container.js

```diff
@@ -87,6 +88,7 @@ function mapStateToProps(state) {
     allAccountsOnNetworkAreEmpty: getAllAccountsOnNetworkAreEmpty(state),
     isTestNet: getIsTestnet(state),
     showExtensionInFullSizeView: getShowExtensionInFullSizeView(state),
+    stxOptIn: getSmartTransactionsOptInStatus(state),
```

Comment:
There's lots of references to `stx` throughout the UI, but we many other long names so would `smartTransactionsEnabled` be more explicit and readable?

Reply:
Sure. Just to clarify `smartTransactionsEnabled` means the STX feature is enabled in UI (e.g. a user is on Ethereum Mainnet, health check worked, etc.) and `smartTransactionsOptInStatus` says if a user turned it on or off.